week_4_analytics_engineering/homework.md

```diff
@@ -0,0 +1,27 @@
+## Week 4 Homework - WIP
+
+In this homework, we'll enhance the alreeady presented dbt project using the already loaded Taxi data from 2019 and 2020 in our DWH.
+
+In this homework, we will:
+
+* Build a source table
+* Build a fact table
+* Create a dashboard 
+
+If you don't have access to GCP, you can do this locally using the ingested data from your Postgres database
+instead. If you have access to GCP, you don't need to do it for local Postgres -
+only if you want.
+
+
+## Question 1: (1 point)
+
+
+
+## Question 2: (1 point)
+
+## Submitting the solutions
+
+* Form for submitting: TBA
+* You can submit your homework multiple times. In this case, only the last submission will be used. 
+
+Deadline: February 14, 22:00 CET 
```

Comment:
Hey @sejalv looks like you need to update your branch, specially because I already changed this deadline in master

Reply:
Don't know why my IDE isn't pulling the changes from main :/ But I'll take care of it

Reply:
Done